actionpack/lib/action_controller/metal/renderers.rb

```diff
@@ -25,14 +25,6 @@ module Renderers
       self._renderers = Set.new.freeze
     end
 
-    module ClassMethods
-      def use_renderers(*args)
-        renderers = _renderers + args
-        self._renderers = renderers.freeze
-      end
-      alias use_renderer use_renderers
-    end
-
```

Comment:
This should probably be changed to a deprecation since I see it has been used in Rails apps such as [thoughtbot's copycopter](https://github.com/copycopter/copycopter-server/blob/d3607c4a032357d121148fee278cba44f87f41e9/app/controllers/api/v2/base_controller.rb#L8)